lib/active_merchant/billing/gateways/element.rb

```diff
@@ -200,7 +207,7 @@ def add_transaction(xml, money, options = {})
           xml.TransactionID options[:trans_id] if options[:trans_id]
           xml.TransactionAmount amount(money.to_i) if money
           xml.MarketCode market_code(money, options) if options[:market_code] || money
-          xml.ReferenceNumber options[:order_id] || SecureRandom.hex(20)
+          xml.ReferenceNumber options[:order_id][0, 50] || SecureRandom.hex(20)
```

Comment:
I'm actually not sure what this is trying to achieve? It also looks like it's causing a bunch of failures in the unit tests, which is why the CI checks are failing.

Reply:
Hello! 
The purpose of the modify was adjust the reference number according to element documentation requirements, sorry, missed the validations in other tests related with the transaction, but now is corrected.